doc/source/deployment/requirements.rst

```diff
@@ -105,5 +101,7 @@ Network Requirements
     the Deployer with dnsmasq to supply DNS functionality, but this is beyond
     the scope of this preview.
 
+Distributed Virtual Routing (DVR) is not supported in this Technology Preview.
```

Comment:
"Technical Preview"

Reply:
Also we need to state flat network is not supported. could be a different commit.

Reply:
I think the official SUSE term is Technology Preview.